pkg/operator/clientset/versioned/clientset.go

```diff
@@ -3,8 +3,8 @@
 package versioned
 
 import (
-	"fmt"
-	"net/http"
+	fmt "fmt"
+	http "net/http"
```

Comment:
I'm not sure what is changing the generated client-gen code.
Auto-generation update?